### ISSUE TITLE
Staging Review finding: On Continue, error handling is incorrect

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#129391]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ursa Minor
**Product:** Travel Pay Claims
**Feature:** Complex Claims v1

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that meets best practices.]()
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On any expense screen clicking "Continue" using the enter key, focus is not moved to the field in error. This may be confusing for users, especially screen reader users, as there's no indication of the error state of fields within the form nor any type of feedback. Note, the user can use a mouse click to trigger the error states, but focus is not moved. 

Other occurences:
- "Submit claim"

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="512" height="353" alt="Continue with error" src="https://github.com/user-attachments/assets/6c592e7f-c423-4af2-b9e3-3113271b4f67" />

</details>

## Recommended action

Ensure error handling follows VADS, where focus is moved to the first field in error, so that the user is aware of the error

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 3.3.1 Error Identification - Level A](https://www.w3.org/TR/WCAG22/#error-identification)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
